pkg/generators/extension.go

```diff
@@ -108,15 +153,15 @@ func parseExtensions(comments []string) ([]extension, []error) {
 	// Next, generate extensions from "idlTags" (e.g. +listType)
 	tagValues := types.ExtractCommentTags("+", comments)
 	for _, idlTag := range sortedMapKeys(tagValues) {
-		xName, exists := tagToExtension[idlTag]
+		xAttrs, exists := tagToExtension[idlTag]
 		if !exists {
 			continue
 		}
 		values := tagValues[idlTag]
 		e := extension{
-			idlTag: idlTag, // listType
-			xName:  xName,  // x-kubernetes-list-type
-			values: values, // [atomic]
+			idlTag: idlTag,       // listType
+			xName:  xAttrs.xName, // x-kubernetes-list-type
+			values: values,       // [atomic]
 		}
 		if err := e.validateAllowedValues(); err != nil {
```

Comment:
Remove validation in parse?

Reply:
I didn't follow this comment, can you expand? :)

Reply:
Oops, I mean remove `validateAllowedValues` from `parseExtensions`.

This is in reference to the comment on `parseExtensions`:
```Validating extensions is performed separately.```

https://github.com/seans3/kube-openapi/blob/69890351242d7509fc807275114ab78514e62b32/pkg/generators/openapi.go#L444-L448

Because validation actually is separate from parsing now:
https://github.com/seans3/kube-openapi/blob/69890351242d7509fc807275114ab78514e62b32/pkg/generators/extension.go#L128-L132

Reply:
oh, good point. @seans3 is the comment at the top of this function wrong?

Reply:
That's a good catch @julianvmodesto . `validateAllowedValues()` is duplicated. I could have sworn I removed it from the parsing function. As you probably noticed, `parseExtensions()` should only return the parse errors, and the `validateMemberExtensions()` returns the validation errors of well-formed extensions.